gzap/doc/gameplay.md

```diff
@@ -10,6 +10,8 @@ Download the [apworld](../../release/gzdoom.apworld) and add it to Archipelago.
 Download the [latest version of the mod](../../release/gzArchipelago-latest.pk3)
 and add it to your gzdoom load order (it doesn't much matter where).
 
+If you are new to using GZDoom, there a [newbie guide](./gzdoom_newplayers.md) available.
```

Comment:
There isn't a `gzdoom_newplayers.md`, though -- did you forget to add it to the commit? I agree it would be good to have, either here or as a link to something on zdoom.org.